SOURCES/el9/osm2pgsql-replication.patch

```diff
@@ -1,17 +1,29 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 7fbc8ce0..8e666388 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -337,6 +337,5 @@ include(GNUInstallDirs)
+ if (ENABLE_INSTALL)
+     install(TARGETS osm2pgsql DESTINATION bin)
+     install(FILES default.style empty.style DESTINATION share/osm2pgsql)
+-    install(CODE "set(OSM2PGSQL_BINDIR ${CMAKE_INSTALL_FULL_BINDIR})
+-                  configure_file(${PROJECT_SOURCE_DIR}/scripts/osm2pgsql-replication ${CMAKE_INSTALL_FULL_BINDIR}/osm2pgsql-replication)")
++    install(FILES scripts/osm2pgsql-replication DESTINATION bin)
```

Comment:
We might want to stick with `PROGRAMS`, which was [previously used here](https://github.com/openstreetmap/osm2pgsql/commit/b3e94d26ab51003fcecc948b9ba5994999d4a94e) (rather than `FILES`)
```suggestion
+    install(PROGRAMS scripts/osm2pgsql-replication DESTINATION bin)
```

> The PROGRAMS form is identical to the FILES form except that the default permissions for the installed file also include OWNER_EXECUTE, GROUP_EXECUTE, and WORLD_EXECUTE. This form is intended to install programs that are not targets, such as shell scripts. Use the TARGETS form to install targets built within the project.

(https://cmake.org/cmake/help/latest/command/install.html#installing-files)

Reply:
Sure enough:
```
$ rpm -qlvp RPMS/x86_64/osm2pgsql-replication-1.7.1-1.el9.x86_64.rpm 
-rw-r--r--    1 root     root                    22733 Oct 27 20:52 /usr/bin/osm2pgsql-replication
```

Reply:
Much better:
```
$ rpm -qlvp RPMS/x86_64/osm2pgsql-replication-1.7.1-1.el9.x86_64.rpm 
-rwxr-xr-x    1 root     root                    22729 Oct 28 16:59 /usr/bin/osm2pgsql-replication
```